### DISCUSSION TITLE
Having hard time starting a carrer

### DISCUSSION THREAD

**Discussion by atrgouti** [community/community#131079] [Programming Help] [answered]
### Body

Hi, I just graduated and I am looking for an internship. I did not think that it would be this hard to find an internship as a developer. I have good skills, especially in frontend development. I am intermediate to expert in React. I don't care about the money or how much I make; all I want is to work on interesting projects and learn new things that are practical in the real world. Here is my portfolio: [www.atrgouti.me](http://www.atrgouti.me/). If anyone can help or has some tips, I would really, really appreciate it. I didn't come here to ask for help until I tried all other ways but failed. Thank you for your attention.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by asdacosta** [comment, marked as answer]
Hey @atrgouti, nice portfolio.
You can send email requests regarding internship to small local (in your country) companies.
Also, you can try this genuine platforms offering remote and freelance jobs that will present you with hands-on jobs:

- 	 FlexJobs
- 	 We work remotely
- 	 NoDesk.co
- 	 Upwork
- 	 Remotive.io
- 	 TrulyRemote.io
- 	 Working Nomads
- 	 Jobpresso
- 	 Indeed
- 	 Glassdoor

**Reply by Josekariz** [reply]
Amazing resource list.

**Reply by atrgouti** [reply]
okay i will keep trying. Thank you @asdacosta

**Comment by Dawoodoz** [comment]
Add classmates on LinkedIn and see where they work. The best references are people working at the company you apply to, which is how I got my first job. Then just show interest in learning, because school only scratches the surface of what you will learn in a job.

While consultancy can be a way to fill your resumé, be aware of consultancy firms not telling how big cut they take from your salary, because they will also treat you like cattle. Even if you are happy with the money you get, the customer expects you to perform based on how much they paid the consultancy firm, which may be two or three times as much if you look at their public profit margins.
